src/server/game/Spells/SpellEffects.cpp

```diff
@@ -5613,7 +5613,7 @@ void Spell::EffectCreateAreaTrigger(SpellEffIndex /*effIndex*/)
         return;
 
     AreaTrigger* areaTrigger = new AreaTrigger();
-    if (!areaTrigger->CreateAreaTrigger(effectInfo->MiscValue, GetCaster(), nullptr, GetSpellInfo(), destTarget->GetPosition(), m_SpellVisual, m_castId))
+    if (!areaTrigger->CreateAreaTrigger(effectInfo->MiscValue, GetCaster(), nullptr, GetSpellInfo(), destTarget->GetPosition(), GetSpellInfo()->GetDuration(), m_SpellVisual, m_castId))
```

Comment:
Incorrect behavior due to SPELLMOD_DURATION.
Consecration's base duration (DurationEntry->Duration, which is returned by GetSpellInfo()->GetDuration()) is 9000 ms, but learning 3 ranks of http://www.wowhead.com/spell=209218/consecration-in-flame will extend it to 12000 ms. It will cause the periodic dummy to deal damage 12 times, but the visual of consecration fades after 9 seconds